COMRS-35 | Add patient's phone number just bellow Age field

prashantsable1991 opened the pull request.

prashantsable1991 committed:
COMRS-35 | Add patient's phone number just bellow Age field

lauravignoli commented on ui/app/i18n/registration/locale_fr.json:
```diff
@@ -81,5 +81,6 @@
   "SELECT_VALUE_FROM_AUTOCOMPLETE_DEFAULT_MESSAGE": "Veuillez sélectionner une valeur de auto complète",
-  "REGISTRATION_ADDTIONAL_IDENTIFIERS": "Identifiants supplémentaires"
+  "REGISTRATION_ADDTIONAL_IDENTIFIERS": "Identifiants supplémentaires",
+  "REGISTRATION_SEARCH_PHONE_NUMBER": "Phone Number [FR]"
```
@prashantsable1991 Please rename this person attribute key to "PERSON_ATTRIBUTE_TYPE_PHONE_NUMBER"

prashantsable1991 replied:
Pushed the changes as suggested please review once.

prashantsable1991 committed:
COMRS-35 | Add patient's phone number just bellow Age field

lauravignoli commented on ui/app/common/attributeTypes/views/attributeInformation.html:
```diff
@@ -6,69 +6,60 @@
 <div class="field-value" data-ng-switch on="::attribute.format">
     <div data-ng-switch-when="java.lang.String">
         <span data-ng-switch="::isAutoComplete(attribute.name)">
-            <input data-ng-switch-when="true"
```
@prashantsable1991 There's a lot of formatting changes here. Please revert and only commit the code changes you made.

Also please use more descriptive commit messages. You can use a commit title and a commit description. For example for this commit you could have used:

COMRS-35 Rename person attribute and validate its length on registration

REGISTRATION_SEARCH_PHONE_NUMBER was renamed to PERSON_ATTRIBUTE_TYPE_PHONE_NUMBER.

Validation was added for maxLength=9.

lauravignoli committed:
Revert "COMRS-35 | Add patient's phone number just bellow Age field"

This reverts commit c55365765bf40e9a2621badd89aa08262294a0a2.

lauravignoli committed:
COMRS-35 Rename person attribute and validate its length on registration

REGISTRATION_SEARCH_PHONE_NUMBER was renamed to PERSON_ATTRIBUTE_TYPE_PHONE_NUMBER.

Validation was added for maxLength=9.
